gpytorch/variational/mean_field_variational_distribution.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python3
+
+import torch
+from ..lazy import DiagLazyTensor
+from ..distributions import MultivariateNormal
+from ._variational_distribution import _VariationalDistribution
+
+
+class MeanFieldVariationalDistribution(_VariationalDistribution):
+    """
+    A :obj:`~gpytorch.variational._VariationalDistribution` that is defined to be a multivariate normal distribution
+    with a diagonal covariance matrix. This will not be as flexible/expressive as a
+    :obj:`~gpytorch.variational.CholeskyVariationalDistribution`.
+
+    :param int num_inducing_points: Size of the variational distribution. This implies that the variational mean
+        should be this size, and the variational covariance matrix should have this many rows and columns.
+    :param torch.Size batch_shape: (Optional.) Specifies an optional batch size
+        for the variational parameters. This is useful for example when doing additive variational inference.
+    :param float mean_init_std: (default=1e-3) Standard deviation of gaussian noise to add to the mean initialization.
+    """
+    def __init__(self, num_inducing_points, batch_shape=torch.Size([]), mean_init_std=1e-3, **kwargs):
+        super().__init__(num_inducing_points=num_inducing_points, batch_shape=batch_shape, mean_init_std=mean_init_std)
+        mean_init = torch.zeros(num_inducing_points)
+        covar_init = torch.ones(num_inducing_points)
+        mean_init = mean_init.repeat(*batch_shape, 1)
+        covar_init = covar_init.repeat(*batch_shape, 1)
+
+        self.register_parameter(name="variational_mean", parameter=torch.nn.Parameter(mean_init))
+        self.register_parameter(name="variational_stddev", parameter=torch.nn.Parameter(covar_init))
```

Comment:
Let’s consider adding a `getter` that returns the absolute value of these. I can already see the “variational inference is learning negative standard deviations” github issues now :-).

Reply:
Yes good point!

Reply:
Might make the getter for the variances directly to avoid an unnecessary .abs.pow(2) when just .pow(2) will suffice.